common/djangoapps/pipeline_mako/templates/static_content.html

```diff
@@ -132,3 +133,11 @@
 <%def name="show_language_selector()"><%
     return settings.FEATURES.get('SHOW_LANGUAGE_SELECTOR', False)
 %></%def>
+
+<%def name="is_user_authenticated()"><%
```

Comment:
I don't think we should write a separate method here when we can do it like this in templates `user.is_authenticated()`